recipes/nspyre/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "nspyre" %}
+{% set version = "0.3.0rc1" %}
```

Comment:
has there ever been a non-rc release?

Reply:
This has been an internal project that we are publicly releasing for the first time. It's currently listed as rc because we were trying to make sure all the publishing works correctly (PyPI as well) because removing the rc label. It looks like we should be able to release 0.3.0 tonight

Reply:
probably best to wait then

Reply:
@scopatz v0.3.0 is now released, this should be good to merge